test/unit/sanitiser/suggest.js

```diff
@@ -160,8 +160,9 @@ module.exports.tests.sanitize_layers = function(test, common) {
   });
   test('invalid layer', function(t) {
     sanitize({ layers: 'test_layer', input: 'test', lat: 0, lon: 0 }, function( err, clean ){
-      var msg = 'invalid param \'layer\': must be one or more of geoname,osmnode,osmway,admin0,admin1,admin2,neighborhood,poi,admin';
```

Comment:
I changed the way this test runs in an attempt to make it less brittle when new layers are added.